From 23d281e11cb924b12508bf9afc9bb1955543d29f Mon Sep 17 00:00:00 2001
From: Alejandro Martinez Ruiz <alex@nowcomputing.com>
Date: Thu, 24 Jan 2013 21:02:14 +0100
Subject: [PATCH 04/12] f2fs: backport QSTR_INIT in namei

The upstream commit introducing QSTR_INIT is 26fe575028703948880fce4355a210c76bb0536e.

Signed-off-by: Alejandro Martinez Ruiz <alex@nowcomputing.com>
---
 fs/f2fs/namei.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/f2fs/namei.c b/fs/f2fs/namei.c
index 1a49b88..1d402e6 100644
--- a/fs/f2fs/namei.c
+++ b/fs/f2fs/namei.c
@@ -183,7 +183,7 @@ out:
 
 struct dentry *f2fs_get_parent(struct dentry *child)
 {
-	struct qstr dotdot = QSTR_INIT("..", 2);
+	struct qstr dotdot = { .name = "..", .len = 2 };
 	unsigned long ino = f2fs_inode_by_name(child->d_inode, &dotdot);
 	if (!ino)
 		return ERR_PTR(-ENOENT);
-- 
1.8.1.rc1.17.g75ed918

